docs/network-spec/connection-manager.tex

```diff
@@ -950,11 +950,34 @@ \subsubsection{\Prune{} transitions}
 \connmngr{} will check if the server hard limit was exceeded. If that
 happened, the \connmngr{} will reset an arbitrary connection (with some preference).
 
-We prefer to reset an inbound connection rather than close an outbound
+There are actually two more transitions that can lead to exceeding server hard limit:
+\begin{itemize}
+  \item \OutboundStateDupTau{} to \InboundStateDup{} (via \DemotedToColdDupLoc{})
+  \item \OutboundStateDupTau{} to \DuplexState{} (via \PromotedToWarmDupRem{})
+\end{itemize}
```

Comment:
I think it's better to change thee paragraph above and itemize (as you do here) all three transitions which do pruning in one place.